### PR TITLE
Update activity ui carousel

### LINKED ISSUE #947: Carousel image for Activity UI
At launch, the activity UI (prototype) will be accessible via the carousel. What is needed:

- [x] Order: second in the carousel
- [x] Image: 
activity_ui_carousel

- [x] Alt tag: New crowdsourcing browse interface. A row of pages volunteers can choose from to transcribe"

- [x] Call to action msg: 
> Find your perfect page
> Log in or register to transcribe and review in our new way of browsing

- [x] URL to `/act`

### PR DESCRIPTION
**Pull request by rstorey**
Closes #947

### CODE REVIEW

**Comment by acdha** [comment]
The image should be 1200x480 but it appears to be 1200x481.

File: concordia/static/scss/base.scss [hunk 1]
```diff
@@ -613,6 +613,7 @@ ul.nav-secondary {
 
 .view-homepage .carousel-indicators .active {
     background: url('data:image/svg+xml;charset=utf8,%3Csvg%20height%3D%2212%22%20viewBox%3D%220%200%2012%2012%22%20width%3D%2212%22%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%3E%3Ccircle%20cx%3D%226%22%20cy%3D%226%22%20fill%3D%22%23F05129%22%20fill-rule%3D%22evenodd%22%20r%3D%225.5%22%20stroke%3D%22%23fff%22/%3E%3C/svg%3E');
+    background-repeat: no-repeat;
```
Comment (acdha) [inline concordia/static/scss/base.scss]: This shouldn't have been necessary. I wonder what changed to break the consistent size for the indicators — it's in master but not release currently.

Comment (rstorey) [reply concordia/static/scss/base.scss]: I was wondering the same thing.

**Comment by rstorey** [comment]
> I was wondering about the duplication for the slide based on the NEW_CAROUSEL branch — the idea being that we always want this one to be second but not first or last?

The request was that the new slide be second - I assumed that means second after Mem Day Challenge or second after Whitman. I believe the Mem Day challenge slide will be turned off on the same day that this launches.

File: concordia/static/scss/base.scss (hunk 1 above)
Comment (acdha) [reply concordia/static/scss/base.scss]: Looks like it was a change added in Bootstrap 4.2 to improve usability by making the click targets taller with a 10px transparent border: https://github.com/twbs/bootstrap/pull/26996

We used 4.1.3 until https://github.com/LibraryOfCongress/concordia/commit/5f9efa090984022b120c2117b95f905b08121f3b

**Pull request merged**